qiskit/utils/quantum_instance.py

```diff
@@ -32,6 +32,8 @@
     is_aer_qasm,
     is_basicaer_provider,
     support_backend_options,
+    _get_backend_provider,
+    _get_backend_interface_version,
```

Comment:
If we're importing them elsewhere, they kind of ought to be public.

Reply:
It's mostly so we don't have to backport a new public interface. We can promote it on main if we want